action_server/src/sema4ai/action_server/_download_rcc.py

```diff
@@ -47,7 +41,30 @@ def download_rcc(
 
     rcc_path.parent.mkdir(parents=True, exist_ok=True)
 
-    rcc_url = RCC_URLS[system or platform.system()]
+    machine = platform.machine()
+    is_64 = not machine or "64" in machine
+
+    if sys.platform == "win32":
+        if is_64:
+            relative_path = "/windows64/rcc.exe"
+        else:
+            relative_path = "/windows32/rcc.exe"
```

Comment:
..same here dropping the 32bit makes this cleaner.

Reply:
Ok, raising exception in this case to make it clear.